site/src/util/workspace.test.ts

```diff
@@ -56,7 +56,7 @@ describe("util > workspace", () => {
       [
         "2022-06-02T10:56:20-04:00",
         {
-          deadline: "2022-06-02T16:26:20Z",
+          deadline: "2022-06-02T18:56:20Z",
```

Comment:
**Review**:

It was previously 90 minutes, now it's 4 hours. So this adds 2.5 hours.